admin-dev/themes/new-theme/js/pages/order/create/customer-renderer.js

```diff
@@ -48,6 +49,8 @@ export default class CustomerRenderer {
 
     if (foundCustomers.length === 0) {
       this._showNotFoundCustomers();
+      this._hideCheckoutHistoryBlock()
+      CreateOrderPage.hideCartInfo();
```

Comment:
I think @PierreRambaud will disagree with that. It's breaking the hierarchy.

Right now we have `CreateOrderPage` that has dependency `CustomerManager` that has dependency `CustomerRenderer`. We cannot have CustomerRenderer call CreateOrderPage, it's a circular dependency and it's very dangerous.

Can we instead emit an event ?

Reply:
Yea I agree :smile: